cmd/cmd.go

```diff
@@ -12,6 +12,7 @@
 // WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
 // See the License for the specific language governing permissions and
 // limitations under the License.
+
```

Comment:
Thank you for your contribution, I checked golang's documentation and haven't found any instructions about having to have a blank line after the license header file. 

Can you explain the need for this modification?